src/frontend/js/components/LtiConsumer/index.tsx

```diff
@@ -19,7 +21,12 @@ const LtiConsumer = ({ id }: LtiConsumerProps) => {
   useAsyncEffect(async () => {
     await fetch(`/api/v1.0/plugins/lti-consumer/${id}/context/`)
       .then(checkResponseStatus)
-      .then(setContext)
+      .then((ltiContext: LtiConsumerContext) =>
+        setContext({
+          ...ltiContext,
+          ...(user && { lis_person_sourcedid: user.username }),
```

Comment:
maybe add a test if easy?

Reply:
Yes of course, I just quickly put the code on github.